README.md

```diff
@@ -122,6 +128,7 @@ Tested platforms include:
 
 -   Fedora 28+
 -   Ubuntu 18.04+
+-   Arch 6.2.2-zen2-1-zen+
```

Comment:
Although not documented, but tested platforms are those inside which I manually tested the resulting AppImage, using Docker containers.

So did you test it too?